ruby-challenges.rb

```diff
@@ -10,18 +10,79 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+#Pseudocode:
+#Create a function called 'contains_letter' and pass an array and a letter as parameters
+#Use 'select' method to filter through the array
+#Use conditional statement with '.includes?' method to see if the word(s) in the array contains the chosen letter
+#Return all words that meet that condition in an array
+
+def contains_letter (array, letter)
+    array.select do |value|
+        if value.include?(letter)
+            value
+        end
+    end
+end
+
```

Comment:
Nice work! Quick note: `.select` already has a built in `if` statement, so no need to add one within the block!